assets/js/theme/global/quick-view.js

```diff
@@ -24,8 +24,8 @@ export default function (context) {
             const $carousel = modal.$content.find('[data-slick]');
 
             if ($carousel.length) {
-                $carousel.on('init', onCarouselChange);
-                $carousel.on('afterChange', onCarouselChange);
+                $carousel.on('init afterChange', $carousel, onCarouselChange);
+                $carousel.on('click', '.slick-arrow', $carousel, onCarouselClick);
```

Comment:
Do we need to add dot buttons?

Reply:
there are no dots in modal slider. Only arrows.